storage/integration_test.go

```diff
@@ -637,6 +637,7 @@ func TestIntegration_DoNotDetectDirectConnectivityWhenDisabled(t *testing.T) {
 
 func TestIntegration_MetricsEnablement(t *testing.T) {
 	ctx := skipHTTP("grpc only test")
+	ctx = skipExtraReadAPIs(ctx, "no reads in test")
 	mr := metric.NewManualReader()
```

Comment:
Ah, this guy being out of the multiTransportTest is probably the culprit, but yes we should skip anyway. Thanks Cathy!